### DISCUSSION TITLE
Would it be possible to make a version of this for gitlab?

### DISCUSSION THREAD

**Discussion by Theartbug** [ArtiomTr/jest-coverage-report-action#145] [Ideas]
This tool looks fantastic but also seems pretty linked into github. Is there any consideration for expanding its usage into gitlab CI/CD?

**Comment by ArtiomTr** [comment]
Hello @Theartbug! 

I think that nothing is impossible for open source projects :smile: 

I googled and found [`@gitbeaker/node`](https://www.npmjs.com/package/@gitbeaker/node), which might do all necessary stuff to expand this action to GitLab api. However, I have never worked with GitLab pipelines, so I can't predict when this will be implemented.

To sum up, this idea sound really interesting. However, this is not our primary issue for now. But, if there are people who want to contribute to this project - feel free to open PR, or just ask for some help regarding this issue. 

P.S. I will create an issue from this discussion, so all future information would be there.
